### ISSUE TITLE
Translations needed

### ISSUE THREAD

**Issue by TyKonKet** [TeamFSIModding/FS17_Goweil_LT_Master#1]
Hi @mieperDE, @naej56, @Gonimy-Vetrom, @Dzi4d3k , @FarmingVidsHD, @Demers94, @oekobauer, @ItZzShady, @PromGames, @Anonymous-any, @KITT3000, @leifege640, @n0tr3adY, @Vanquish081, @joshvzeyl, @wieselflinked, @gorgona2, @baleBaron.
This is the repository of our project for the **Mod Contest 2017**, it's almost at the end but we would like to have as much as possible different translations so we are asking for your help.
We need new translations but if you think you can improve existing ones (especially EN), please update them.
If you will help us, don't forget to leve your credits 😄.
Hope you'll like our mod, sorry for the disturbance, have a nice day.

**Comment by Dzi4d3k** [comment]
Hi @TyKonKet, because EN translation is not quite obvious for me for some text and mod is not working, which is quite understandable. I would  like you to give some explanation on following description, because  , because as allays, I would like to make meaningful and contextual translation rather than direct.

- Dynamic density chamber - is that mean that you can chose (to buy) functionality to be able to produce bales with different compression level? Is this is automatically change or manually? If yes then better would be better to call it "Variable compression chamber"?
- Open and fade bale - how it works?
- Silage Additives is used to get higher resolution - is "resolution" used in context of decomposition (decay)?

**Comment by leifege640** [comment]
Hello TyKonKet,

I am honored that you would offer this opportunity to me, however I am
afraid that I must decline. I am about to leave for a custom harvesting job
in a week. Thus between packing and saying goodbye to friends for the next
6 months I feel I won't have enough time to help you out.

Again thank you for your request. If you need my assistance later in the
year feel free to ask.

Sincerely,

Leif Ege

**Comment by fcelsa** [comment]
From my side, 
1) "Variable compression chamber" it's ok, compression level is an option in shop, and when work, various compression can be set with a key IMPLEMENT_EXTRA3 this result in more or less liters per bale.
2) Open and fade bale mean when you walk and you go in front of bales, a little cutter hud appear togheter with default hand, also an information text appear in the bale, at this point you can use InputBinding.ACTIVATE_OBJECT to open the bale, then the material tip to ground and bale vanishes.
3) From the game side, fill machine with additive mean a more productivity, I don't remember how much, now on the fly, but i think around 20% better productions; from the real world side, I think additive usage it's useful to a better fermentage and better animals feed product.
You are free to translate "resolution", I took this phrase from Goeweil web site :)
Maybe TyKonKet can add more other explanetions.

**Comment by Dzi4d3k** [comment]
@fcelsa
Thank you, I think at this point is enough for me to understand functionalities.

**Comment by Dzi4d3k** [comment]
I suggest all to look at brochures for most accurate translations:
https://www.goeweil.com/index.php/en/mediacenter-engl/mediacenter-pdf-engl

**Comment by joshvzeyl** [comment]
i added the dutch translation as pull request for you guys and i looked at the pictures in the brochure  and i think that the mod is going to be amazing

**Comment by n0tr3adY** [comment]
Hi,

Count on me, i'll make the hungarian translation within few days ;)

**Comment by Vanquish081** [comment]
I will help Promgames, who has problems with Spanish, is more English than Spanish xD

**Comment by PromGames** [comment]
On vie, 26 may 2017 at 20:16 Vanquish081 <notifications@github.com> wrote:

> I will help Promgames, who has problems with Spanish, is more English than
> Spanish xD
>

I will make the first pass and he will do the fine tune, a usually.

Then expect two pull requests

Prom

**Comment by Dzi4d3k** [comment]
`<text name="GLTM_HL_TEXT_02_00"                     text="Position tractor and machine on plain terrain and turn little bit to the left, then you can put machine in work position with $input_IMPLEMENT_EXTRA2; " />`

`<text name="GLTM_HL_TEXT_02_01"                     text="Go down to machine, open right door with $input_TOGGLE_COVER; to access control panel, put down support with $input_IMPLEMENT_EXTRA4; and set machine in work position with $input_IMPLEMENT_EXTRA; finally set down bales chute with $input_IMPLEMENT_EXTRA2; " />`

I'm slightly confused, work position have to be enable in two steps?

**Comment by fcelsa** [comment]
Yes this is right, but it's possible to describe in other words, first of all you need to put tractor in work positions, then the machine, it's not easy to explain better due to my poor English ;)

**Comment by Dzi4d3k** [comment]
I suppose first one is changing position of the hitch to side, right?

**Comment by TyKonKet** [comment]
To put the machine in work position you must follow 5 steps:
1) X inside the tractor to open at 30° the drawbar
2) Down the tractor open the side door
3) Down the tractor lower the supports
4) Down the tractor open the back part of machine
5) Down the tractor open the bale slide

**Comment by fcelsa** [comment]
Yes is it

**Comment by Gonimy-Vetrom** [comment]
Can you make a video with all the features?

**Comment by fcelsa** [comment]
It's on the way, I hope tomorrow can put a small video for functions.

**Comment by Dzi4d3k** [comment]
Now I'm confused even more. When I made mod from Git it game load normally, but when I translated ln10_pl, and wanted to verify texts in game I got following error in log:
`Error: Running LUA method 'loadMapFinished'.
C:/Documents/My Games/FarmingSimulator2017/mods/FS17_Goweil_LT_Master/scripts/helpers/Utils.lua:85: attempt to index local 'actionData' (a nil value)`

And game lead indefinitely.
Not sure if this is intended, but I think I would let you know,

**Comment by fcelsa** [comment]
You forgot semicolon at the end of key definitions in helpline text, for example 
 $input_IMPLEMENT_EXTRA4;

**Comment by Dzi4d3k** [comment]
ok, I'll fix that

**Comment by Dzi4d3k** [comment]
I see that after translation, some help pages would need scrolling because text is going out of window. Guys form Realismus Modding done this in Seasons.

**Comment by Dzi4d3k** [comment]
Additionally plates names in shop are not available for translation.

**Comment by TyKonKet** [comment]
What do you mean with "plates"?

**Comment by Dzi4d3k** [comment]
Sorry, typo. Should be pallets with net, foil and additive. They have only English names.

**Comment by TyKonKet** [comment]
You must translate them from their xml in the store data section.

**Comment by fcelsa** [comment]
> I see that after translation, some help pages would need scrolling because text is going out of window. Guys form Realismus Modding done this in Seasons.

In the default ingame helpline, scroll bar is only for chapter, not for text, Even the Seasons mod. 
You needs to set length of text as better possible to fit in the space available.

**Comment by Dzi4d3k** [comment]
Right, I'll check what I can do.

**Comment by Dzi4d3k** [comment]
Problematic is only the page: "Net, wrap foil, Additive"
Maybe picture on can be little smaller, because there is no room for just one line.

**Comment by fcelsa** [comment]
Yes I need to replace picture of pallets additive, net, foil, and probably will replace with half size picture.

**Comment by Gonimy-Vetrom** [comment]
Setting the machine to the working position. https://youtu.be/y3G8XLNuqQM?t=3m58s

**Comment by Vanquish081** [comment]
We not only know translate, we also know how to test the mod ... Killercrok only knows how to test the mod? ¬_¬

**Comment by fcelsa** [comment]
Yes, I know ;)

**Comment by Vanquish081** [comment]
Ok... your welcome...

**Comment by Dzi4d3k** [comment]
Well, hope not too much time left for public availability and I suppose that there will be next versions which can incorporated required fixes, also these translation related.
